### PR TITLE
chore: document sidebarTitle frontmatter

### PR DESCRIPTION
**Pull request by arno-fukuda**
## Why:
Documentation for sidebarTitle per https://github.com/shuding/nextra/issues/3475

## Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View
      deployment** link in this PR's timeline (this link will be available after
      opening the PR).

### CODE REVIEW

File: docs/pages/docs/docs-theme/theme-configuration.mdx [hunk 1]
```diff
@@ -403,6 +403,22 @@ export default {
...
+---
+sidebarTitle: Getting Started 🚀
+---
+```
+The priority of the sidebar title is as follows:
+
+1. `title` property from `_meta.js` file
+1. Frontmatter `sidebarTitle`
+1. Frontmatter `title`
+1. Filename of the page formatted with with [Title](https://title.sh)
```
Comment (dimaMachina) [inline docs/pages/docs/docs-theme/theme-configuration.mdx:420]:
```suggestion
1. Filename of the page formatted with [Title](https://title.sh) 
```

File: docs/pages/docs/docs-theme/theme-configuration.mdx [hunk 2]
```diff
@@ -403,6 +403,23 @@ export default {
...
+---
+sidebarTitle: Getting Started 🚀
+---
+```
+
+The priority of the sidebar title is as follows:
+
+1. `title` property from `_meta.js` file
+1. Frontmatter `sidebarTitle`
+1. Frontmatter `title`
```
Comment (dimaMachina) [inline docs/pages/docs/docs-theme/theme-configuration.mdx:420]:
```suggestion
In addition, you can customize the sidebar title using the `sidebarTitle` property in your front matter:

```md filename="pages/getting-started.mdx"
---
sidebarTitle: Getting Started 🚀
---
```

The priority of the sidebar title is as follows:

1. `title` property from `_meta.js` file
1. Front matter `sidebarTitle`
1. Front matter `title`
```

let's stays with front matter https://github.com/jekyll/jekyll/issues/2396#issuecomment-50858507

File: docs/pages/docs/docs-theme/theme-configuration.mdx [hunk 3]
```diff
@@ -403,6 +403,23 @@ export default {
 
 <Screenshot src={sidebarTitleComponentImage} alt="Customized Sidebar" />
 
+#### Customize Sidebar with Frontmatter
+
+In addition, you can customize the sidebar title using the `sidebarTitle` property in your front matter:
+
+```md filename="pages/getting-started.mdx"
```
Comment (dimaMachina) [inline docs/pages/docs/docs-theme/theme-configuration.mdx:410]:
````suggestion
```mdx filename="pages/getting-started.mdx"
````

to have mdx icon

File: docs/pages/docs/docs-theme/theme-configuration.mdx [hunk 4]
```diff
@@ -403,6 +403,23 @@ export default {
 
 <Screenshot src={sidebarTitleComponentImage} alt="Customized Sidebar" />
 
+#### Customize Sidebar with Frontmatter
```
Comment (dimaMachina) [inline docs/pages/docs/docs-theme/theme-configuration.mdx:406]:
```suggestion
### Customize Sidebar with Frontmatter
```

Comment (dimaMachina) [reply docs/pages/docs/docs-theme/theme-configuration.mdx:406]: looks better like this

**Review by dimaMachina — APPROVED** [review]
thanks for contribution on improving Nextra docs 🎉

**Comment by arno-fukuda** [comment]
> thanks for contribution on improving Nextra docs 🎉

pleasure, hope to help out even more

**Comment by dimaMachina** [comment]
always appreciated 😍

**Pull request merged**